server/db/migrations/patch-2-3.sql

```diff
@@ -1,2 +0,0 @@
--- used for batch reindexing based on creation time
-CREATE INDEX user_pages_created_at ON user_pages (created_at);
```

Comment:
you guessed it! I'm avoiding the horror of "hmm, how to migrate the data" as long as possible